src/main/java/com/mentalhealthapi/mentalhealthapi/dto/Blog.java

```diff
@@ -4,8 +4,8 @@
 
 public @Data class Blog {
     private int id;
-    private int userId;
     // or private int User; firebase auth?
+    private int userId;
```

Comment:
One comment seemed to be below the code it was intended for and changing its position would be more inline with the comment standards of Baeldung. https://www.baeldung.com/java-clean-code#8-code-comments

Reply:
Is the inline comment even needed?  JavaDoc is always a good idea, but inline comments are generally minimized...  and only used when the code is not adequately self-describing.

Reply:
I had this comment here to remind myself of firebase. At the time, I was trying to compile a list of ways we can tackle the challenge of auth. I removed this comment recently.